### PR TITLE
Update commons-codec, gson, and cxf-rt-rs-client dependencies

### PR DESCRIPTION
**Pull request by gl-johnson**
### What does this PR do?
Updates the following depedencies due to vulnerabilities identified in Snyk scans:

- commons-codec:commons-codec
- com.google.code.gson:gson
- org.apache.cxf:cxf-rt-rs-client

### CODE REVIEW

**Review by szh — APPROVED** [review]
LGTM! Does this need review from @andytinkham to confirm it fixed the Snyk warnings?

**Review by andytinkham — APPROVED** [review]
LGTM to me too. I can't see the full Snyk scan until it's merged. Will double check at that point.

**Comment by andytinkham** [comment]
Confirmed. Snyk shows no current vulnerabilities in the pom.xml file after merging

**Pull request merged**
